momaland/learning/iql/iql.py

```diff
@@ -0,0 +1,273 @@
+"""Implementation of stateless independent Q-learners. Implemented for the multi-objective congestion game and beach domain."""
+
+import random
+
+import numpy as np
+from gymnasium.spaces import Discrete
+
+from momaland.envs.beach_domain.beach_domain import (
+    MOBeachDomain,
+    _global_capacity_reward,
+    _global_mixture_reward,
+)
+
+
+# from momaland.envs.congestion_game import mocongestion_v0 as CongestionGame
+
+
+class TabularMOBeachDomainWrapper(MOBeachDomain):
```

Comment:
Might be wise to move this wrapper to another file to have this file focus on IQL implementation.

Reply:
Moved it to a different file, makes more sense indeed to have the IQL implementation on its own.